tests/classes/core/PKPRequestTest.php

```diff
@@ -254,7 +254,7 @@ public function testGetServerHostWithHostname() {
 		$_SERVER = array(
```

Comment:
I think the point of this test would now be to ensure that we *aren't* picking up the (provided) HOSTNAME from the `$_SERVER` environment.